docs/deployment/README.md

```diff
@@ -190,58 +198,60 @@ If you're re-deploying based on an existing configuration you can skip this conf
     subnets:
       - '{{AWS_SUBNET}}'        # this has to be set in .env
 
-  ecs:
+  ecs:                          # Required
     instanceType: t2.micro
     desiredInstances: 0
     availabilityZone: <subnet-id-zone>
     amiid: <some-ami-id>
 
-  # Or can specify a different bucket for the system_bucket
+  # Required. You can specify a different bucket for the system_bucket
   system_bucket: '{{buckets.internal.name}}'
 
-  buckets:                              # Should be the same as in IAMs
+  # Required. Should be the same as in IAM deployment config.yml
+  buckets:
     internal:
         name: dev-internal
         type: internal
 
+  # Required.
   iams:
-    ecsRoleArn: arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/<iams-prefix>-ecs
-    lambdaApiGatewayRoleArn: arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/<iams-prefix>-lambda-api-gateway
-    lambdaProcessingRoleArn: arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/<iams-prefix>-lambda-processing
-    stepRoleArn: arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/<iams-prefix>-steprole
-    instanceProfile: arn:aws:iam::{{AWS_ACCOUNT_ID}}:instance-profile/<iams-prefix>-ecs
+    ecsRoleArn: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/<iams-prefix>-ecs'
+    lambdaApiGatewayRoleArn: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/<iams-prefix>-lambda-api-gateway'
+    lambdaProcessingRoleArn: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/<iams-prefix>-lambda-processing'
+    stepRoleArn: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/<iams-prefix>-steprole'
+    instanceProfile: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:instance-profile/<iams-prefix>-ecs'
     distributionRoleArn: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/<iams-prefix>-distribution-api-lambda'
     scalingRoleArn: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/<iams-prefix>-scaling-role'
     migrationRoleArn: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/<iams-prefix>-migration-processing'
 
-  urs_url: https://uat.urs.earthdata.nasa.gov/ #make sure to include the trailing slash
+  # Optional
+  urs_url: https://uat.urs.earthdata.nasa.gov/ # make sure to include the trailing slash
 
-  # if not specified the value of the apigateway backend endpoint is used
-  # api_backend_url: https://apigateway-url-to-api-backend/ #make sure to include the trailing slash
+  # if not specified, the value of the API gateway backend endpoint is used
+  # api_backend_url: https://apigateway-url-to-api-backend/ # make sure to include the trailing slash
 
-  # if not specified the value of the apigateway dist url is used
-  # api_distribution_url: https://apigateway-url-to-distribution-app/ #make sure to include the trailing slash
+  # if not specified, the value of the API gateway distribution endpoint is used
+  # api_distribution_url: https://apigateway-url-to-distribution-app/ # make sure to include the trailing slash
 
-  # URS users who should have access to the dashboard application and use the
-  # distribution API to access protected files.
+  # Required. URS users who should have access to the dashboard application.
```

Comment:
This isn't really required by the Cloudformation template, but if they don't specify it, then they aren't giving access to any of their users. So it's functionally required.